### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by brucearmstrong**
Update Swagger doc links which have changed.

### CODE REVIEW

File: android/MobiledgeXSDKDemo/README.md [hunk 1]
```diff
@@ -28,22 +28,23 @@ You can also choose this cloudlet to be used as the **Edge** server for Face Det
...
+This will call the GetAppInstListAPI to find everywhere our backend is running, and will draw a cloudlet icon for every location found.
 
 #### Verify Location
-This will call the [VerifyLocationAPI](http://swagger.mobiledgex.net/client/#operation/VerifyLocation "VerifyLocation") with our current GPS coordinates to verify that the mobile device is where it claims to be. If successful, the mobile phone icon is green showing the accuracy information briefly. If location verification fails, the icon will be red, displaying the result code temporarily. Tapping the icon will show the result code of the call.
+This will call the [VerifyLocationAPI](https://swagger.mobiledgex.net/#section/Edge-SDK-Android/VerifyLocation "VerifyLocation") with our current GPS coordinates to verify that the mobile device is where it claims to be. If successful, the mobile phone icon is green showing the accuracy information briefly. If location verification fails, the icon will be red, displaying the result code temporarily. Tapping the icon will show the result code of the call.
```
Comment (vmohanx) [inline android/MobiledgeXSDKDemo/README.md:38]: i think you wanted these links to point to https://api.mobiledgex.net/ ? I am getting a 404 from swagger.mobiledgex.net

File: android/MobiledgeXSDKDemo/README.md [hunk 2]
```diff
@@ -28,22 +28,23 @@ You can also choose this cloudlet to be used as the **Edge** server for Face Det
 
 ### SDK Calls
 #### Register Client
-This will call the [RegisterClientAPI](http://swagger.mobiledgex.net/client/#operation/RegisterClient "RegisterClient") with information that identifies this app's backend software. The session cookie is shown to verify that the call was successful.
+This will call the [RegisterClientAPI](https://
```
Comment (vmohanx) [inline android/MobiledgeXSDKDemo/README.md:31]: link here looks broken

File: android/MobiledgeXSDKDemo/README.md [hunk 3]
```diff
@@ -28,22 +28,23 @@ You can also choose this cloudlet to be used as the **Edge** server for Face Det
...
+.mobiledgex.net/#section/Edge-SDK-Android/RegisterClient "RegisterClient") with information that identifies this app's backend software. The session cookie is shown to verify that the call was successful.
```
Comment (vmohanx) [inline android/MobiledgeXSDKDemo/README.md:32]: maybe also add that the information is the org_name, app_name, app_version

**Comment by brucearmstrong** [comment]
I was trying to reply to a previous comment, but it looks like it disappeared. At first I was confused why the same docs existed at 2 different URLs, but then I realized that swagger.mobiledgex.net and api.mobiledgex.net resolve to the same IP address.

@vmohanx, which URL is preferred?

**Comment by vmohanx** [comment]
@brucearmstrong yeah i deleted comment cause I realized i copied the link wrong. i think either is fine, but developer portal has been pointing to api.mobiledgex.net since i joined so that was the one I have been using.

**Pull request merged**